app/views/users/show.html.erb

```diff
@@ -1 +1,25 @@
-<h1>User details</h1>
\ No newline at end of file
+<div class="user-show-post">
+  <%= render partial: "user_info", locals: {user: @user} %>
+  <div class="bio-post">
+    <div class="border bio">
+      <h3>Bio</h3>
+      <p class="bio-text">
+        <%= @user.bio %>
+      </p>
+    </div>
+
+    <div class="user-posts">
+      <% @user.recent_posts.each do |post| %>
+        <%= link_to user_post_path(@user.id, post.id) do %>
+          <%= render partial: "user_post", locals: {post: post} %>
+        <% end %>
+      <% end %>
+    </div>
+  <div>
+
+  <div class="all-posts">
+      <%= link_to user_posts_path(@user.id) do %>
+        <p class="see-all-posts">See all posts</p>
+      <% end %>
```

Comment:
- [ ] Kindly fix this button, only three recent posts are retrieved when a user clicks the __See all posts__. You can solve this issue by not calling the __recent_posts__ method in the post-controller index action. Use the __recent_posts__ function in the show view to loop through the recent posts just like the __recent_comments__ function has been used in the posts index view.